src/platform/user/authentication/utilities.js

```diff
@@ -8,25 +8,28 @@ export const authnSettings = {
   RETURN_URL: 'authReturnUrl',
 };
 
-const SESSIONS_URI = `${environment.API_URL}/sessions`;
-const sessionTypeUrl = type => `${SESSIONS_URI}/${type}/new`;
+function sessionTypeUrl(type, version = 'v0') {
```

Comment:
@hinzed1127 is this function going to be used by both the new static login page and the existing modal?

Reply:
that's my intent with this first pass. I wanted to share logic more than fragment it where possible. adding a `version` param and then feature flagging it could enable us to separate the 2 endpoints and/or share them down the line if we want the modal/standalone page to co-exist with the same or different endpoints/functionality

Reply:
good to know that the logic is just in one function.  Yeah I'm thinking that we're going to need to add some feature flag logic here that a) checks to see if the feature flag is enabled or b) checks to see if the view requested is the static login page.  If either of those statements are true then render the v1 path